buildenv/jenkins/variables/defaults.yml

```diff
@@ -47,10 +47,10 @@ openj9:
 omr:
   default:
     repoUrl: 'https://github.com/eclipse/omr.git'
-# AdoptOpenJDK Testing repository
-adoptopenjdk:
+# Adoptium AQA Testing repository
+adoptium:
```

Comment:
You're changing the variable name. So you'd have to update equivalent code.
https://github.com/eclipse-openj9/openj9/blob/master/buildenv/jenkins/common/variables-functions.groovy#L1442
In my opinion, I would leave it as `adoptopenjdk`.

Reply:
Thanks @AdamBrousseau . I think this code is used for reading the internal variable file. I will leave it as adoptopenjdk.